src/commands/split_row.rs

```diff
@@ -1,27 +1,40 @@
+use crate::commands::WholeStreamCommand;
 use crate::errors::ShellError;
 use crate::object::{Primitive, Value};
 use crate::prelude::*;
 use log::trace;
 
-pub fn split_row(
-    args: CommandArgs,
-    registry: &CommandRegistry,
-) -> Result<OutputStream, ShellError> {
+pub struct SplitRow;
+
+impl WholeStreamCommand for SplitRow {
+    fn run(
+        &self,
+        args: CommandArgs,
+        registry: &CommandRegistry,
+    ) -> Result<OutputStream, ShellError> {
+        split_row(args, registry)
+    }
+
+    fn name(&self) -> &str {
+        "split-row"
+    }
+
+    fn signature(&self) -> Signature {
+        // TODO: Signature?
+        // TODO: Improve error. Old error had extra info:
+        //
+        //   needs parameter (e.g. split-row ",")
+        Signature::build("split-row").required("delimeter", SyntaxType::Any)
```

Comment:
Signature looks right, though I think it's spelled "delimiter" (I think the same goes for split_column also)